### DISCUSSION TITLE
PvP outposts spawn clipped into terrain (often)

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#14955] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/3ee12ebc-a9a8-45d3-a908-00bb2b8f10ae)
![image](https://github.com/user-attachments/assets/6cf264c2-f13b-42d3-9399-70488e8af641)

It makes view movement and mouse movement all jittery, messes up spectator mode (massive shaking and teleporting), and if you have any moving parts such as elevators/shuttles, screws that up too

### Reproduction steps

1. start outpost deathmatch pvp with a big outpost
2. repeat until eventually it spawns inside terrain
3. may not be an issue with the TWO vanilla outposts, but certainly causes issues with custom ones

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Outposts for pvp matches generate at the center of the level.
Closing discussion as completed.

**Reply by girlyguppy** [reply]
current generator fix when
